CHANGELOG.rst

```diff
@@ -6,6 +6,275 @@
+
+  ``it:cmd:session``
+    A command line session with multiple commands run over time.
+
+  ``it:host:tenancy``
+    A time window where a host was a tenant run by another host.
+
+  ``it:network:type:taxonomy``
+    A taxonomy of network types.
+
+  ``it:software:image:type:taxonomy``
+    A taxonomy of software image types.
+
+  ``it:software:image``
+    The base image used to create a container or OS.
+
+  ``it:storage:mount``
+    A storage volume that has been attached to an image.
+
+  ``it:storage:volume``
+    A physical or logical storage volume that can be attached to a
+    physical/virtual machine or container.
+
+  ``it:storage:volume:type:taxonomy``
+    A taxonomy of storage volume types.
+
+  **New Properties**
+
+  ``biz:listing``
+    The form had the following properties added to it:
+
+    ``count:total``
+      The number of instances for sale.
+
+    ``count:remaining``
+      The current remaining number of instances for sale.
+
+  ``econ:purchase``
+    The form had the following property added to it:
+
+    ``listing``
+      The purchase was made based on the given listing.
+
+  ``it:exec:proc``
+    The form had the following property added to it:
+
+    ``cmd:history``
+      The command history entry which caused this process to be run.
+
+  ``it:exec:query``
+    The form had the following property added to it:
+
+    ``synuser``
+      The synapse user who executed the query.
+
+  ``it:host``
+    The form had the following property added to it:
+
+    ``image``
+      The container image or OS image running on the host.
+
+  ``it:network``
+    The form had the following property added to it:
+
+    ``type``
+      The type of network.
+
+  ``meta:note``
+    The form had the following property added to it:
+
+    ``replyto``
+      The note is a reply to the specified note.
+
+  ``ou:campaign``
+    The form had the following property added to it:
+
+    ``ext:id``
+      An external identifier for the campaign.
+
+  ``ou:org``
+    The form had the following property added to it:
+
+    ``ext:id``
+      An external identifier for the organization.
+
+  ``ou:technique``
+    The form had the following property added to it:
+
+    ``ext:id``
+      An external identifier for the technique.
+
+  ``risk:extortion``
+    The form had the following properties added to it:
+
+    ``paid:price``
+      The total price paid by the target of the extortion.
+
+    ``payments``
+      Payments made from the target to the attacker.
+
+  ``risk:leak``
+    The form had the following properties added to it:
+
+    ``size:count``
+      The number of files included in the leaked data.
+
+    ``size:percent``
+      The total percent of the data leaked.
+
+  ``risk:threat``
+    The form had the following property added to it:
+
+    ``ext:id``
+      An external identifier for the threat.
+
+  **Updated Types**
+
+  ``inet:web:hashtag``
+    Loosen the regex for hashtag validation.
```

Comment:
Do you want mention it supporting unicode now?

Reply:
It's always supported unicode word characters. We only added the middle dot (U+00B7) as an allowed character AFTER the first unicode word character in the hashtag. We could be clear about that.